recipes/patsy/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.4.1" %}
+{% set gh_org = "pydata" %}
+{% set gh_repo = "patsy" %}
+{% set pkg_name = "patsy" %}
+
+
+package:
+  name: {{ pkg_name }}
+  version: {{ version }}
+
+source:
+  fn: {{ pkg_name }}-{{ version }}.zip
+  url: https://pypi.io/packages/source/{{ pkg_name[0] }}/{{ pkg_name }}/{{ pkg_name }}-{{ version }}.zip
+  sha256: dc1cc280045b0e6e50c04706fd1e26d2a00ea400aa112f88e8142f88b0b7d3d4
+
+build:
+  number: 0
+  script:
+    - python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy >=1.4.0
+    - scipy
+    - six
+
+test:
+  imports:
+    - patsy
+
+about:
+  home: https://github.com/{{ gh_org }}/{{ gh_repo }}
+  license:
+    - BSD-2-Clause
+    - PSF2
```

Comment:
Right. This would probably be the better way to do this. I don't know what support exists in `conda-build` or `anaconda-client` for this form, but it would be interesting to know.

Normally what we do is list them in a single string. Here's an [example]( https://github.com/conda-forge/fabio-feedstock/blob/5df95d298920c10c44d613024e04c4308b8218bc/recipe/meta.yaml#L37 ). If that doesn't work, definitely let me know.